starters/next-drupal-starter/__tests__/drupalStateContext.test.jsx

```diff
@@ -0,0 +1,54 @@
+import { suite, describe, afterEach, beforeAll, expect, it, vi } from "vitest";
+import * as utils from "../lib/drupalStateContext";
+
+suite("drupalStateContextUtils", () => {
+  afterEach(() => {
+    vi.restoreAllMocks();
+  });
+
+  describe("makeLocaleStores()", () => {
+    const spy = vi.spyOn(utils, "makeLocaleStores");
+    it("should make a store for each locale", () => {
+      const locales = ["en", "es"];
+      const stores = utils.makeLocaleStores({ locales });
+
+      expect(stores.length).toEqual(2);
+      expect(stores[0].defaultLocale).toEqual("en");
```

Comment:
In my opinion, you should only test one thing for unit test

Reply:
So would you recommend 1 test for length, and another to check default locale for each store, etc?

Reply:
Probably one for length and one to check defaultLocal for both stores, because the length and the defaultLocal looks like 2 different behaviors